substrate/frame/revive/src/wasm/runtime.rs

```diff
@@ -525,7 +525,7 @@ enum CallType {
 	/// Execute another instantiated contract
 	Call { callee_ptr: u32, value_ptr: u32, deposit_ptr: u32, weight: Weight },
 	/// Execute deployed code in the context (storage, account ID, value) of the caller contract
-	DelegateCall { code_hash_ptr: u32 },
+	DelegateCall { address_ptr: u32, deposit_ptr: u32, weight: Weight },
```

Comment:
All three fields should be lifted because they are used in both variants.

Reply:
I'm not sure what you mean

Reply:
do you mean to remove the variants and have a boolean indicating if it's a delegate?

Reply:
@athei tagging you in case you missed this